apps/spark/src/main/java/com/linkedin/openhouse/jobs/scheduler/tasks/TableOperationTask.java

```diff
@@ -22,7 +26,21 @@ protected TableOperationTask(
   protected boolean launchJob() {
     String jobName =
         String.format("%s_%s_%s", getType(), metadata.getDbName(), metadata.getTableName());
-    jobId = jobsClient.launch(jobName, getType(), metadata.getCreator(), getArgs()).orElse(null);
+    jobId =
+        jobsClient
+            .launch(jobName, getType(), metadata.getCreator(), getExecutionProperties(), getArgs())
+            .orElse(null);
     return jobId != null;
   }
+
+  protected Map<String, String> getExecutionProperties() {
+    final String propertyPrefix = MAINTENANCE_PROPERTY_PREFIX + getType().name() + ".";
```

Comment:
Wouldn't this require duplicating properties in case similar execution confs are needed?
If we want to be able to define specific job props, can we consider a setup with `common` and `jobType` specific property prefixes?

Reply:
I believe maintenance properties are optional and will present for the tables for which global maintenance property override is required. So this method should return empty if there is no such properties. I think jobs type is added after the prefix and that should eliminate the duplicate. Adding some examples of these properties in the PR would be helpful.

Reply:
For one table as well, this case is possible:
`maintenance.OFD.spark.diver.memory=4G
maintenance.SE.spark.diver.memory=4G`

Reply:
Thanks, I was hesitating whether to support type agnostic props or not, but in the end I think ops resource requirements are different for different type, e.g. `OFD.spark.driver.memory=4G` might be reasonable for some tables whereas `SE.spark.driver.memory=2G` is enough for all tables because SE is super lightweight. Handling type-agnostic proper together with typed is kind of involving, so let's not do this for now.

Reply:
Sure. Job specific properties sounds good to me for now. Thanks.

Reply:
Sure, it could be the next evolution depending on the use case.